contracts/core/liquidators/twap-impl/TWAPAuction.sol

```diff
@@ -39,4 +56,357 @@ contract TWAPAuction is TwoAssetPriceBoundedLinearAuction {
         uint256 chunkAuctionPeriod;
         uint256 chunkSize;
     }
+
+    struct TWAPLiquidatorData {
+        uint256 usdChunkSize;
+        uint256 chunkAuctionPeriod;
+    }
+
+    /* ============ Constants ============ */
+
+    // Auction completion buffer assumes completion potentially 2% after fair value when auction started
+    uint256 constant AUCTION_COMPLETION_BUFFER = 2;
```

Comment:
Add explicit `visibility` modifier (e.g. private/public)

Reply:
Just a thought - This assumption could be void if you have auctions that run away from fair value.

Would this be a variable that we'd ever want to change (e.g. via an admin function)?

Reply:
What would cause us to change it? We foresee more volatility coming up? Feels like something we'd only change in an instant and then want to change back again, but don't think that's what we want to do.